astronomer/providers/http/example_dags/example_external_deployment_sensor.py

```diff
@@ -0,0 +1,67 @@
+import os
+from datetime import timedelta
+
+import requests
+from airflow import DAG
+from airflow.hooks.base import BaseHook
+from airflow.utils.log.secrets_masker import mask_secret
+from airflow.utils.timezone import datetime
+
+from astronomer.providers.http.sensors.external_deployment_task import (
+    ExternalDeploymentTaskSensorAsync,
+)
+
+DEPLOYMENT_CONN_ID = os.getenv("ASTRO_DEPLOYMENT_CONN_ID", "http_default")
+EXECUTION_TIMEOUT = int(os.getenv("EXECUTION_TIMEOUT", 6))
+
+default_args = {
+    "execution_timeout": timedelta(hours=EXECUTION_TIMEOUT),
+    "retries": int(os.getenv("DEFAULT_TASK_RETRIES", 2)),
+    "retry_delay": timedelta(seconds=int(os.getenv("DEFAULT_RETRY_DELAY_SECONDS", 60))),
+}
+
+
+def astro_access_token():
```

Comment:
Where is this function being called?

Reply:
I have fixed the example DAG, it was missing some code, added it now